OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -1674,18 +1675,24 @@ def test_browse_filtering_allows_good(client, mockdata, session):
         )
 
         filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Rank</dt>")[1:]
-        assert any(
-            "<dd>{}</dd>".format(job.job_title) in token for token in filter_list
-        )
+        assert any("{}".format(job.job_title) in token for token in filter_list)
+        assert any("<dd>" in token for token in filter_list)
+        assert any("</dd>" in token for token in filter_list)
 
         filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Unit</dt>")[1:]
-        assert any("<dd>{}</dd>".format(unit.descrip) in token for token in filter_list)
+        assert any("{}".format(unit.descrip) in token for token in filter_list)
+        assert any("<dd>" in token for token in filter_list)
+        assert any("</dd>" in token for token in filter_list)
 
         filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Race</dt>")[1:]
-        assert any("<dd>White</dd>" in token for token in filter_list)
+        assert any("White" in token for token in filter_list)
+        assert any("<dd>" in token for token in filter_list)
+        assert any("</dd>" in token for token in filter_list)
 
         filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Gender</dt>")[1:]
-        assert any("<dd>Male</dd>" in token for token in filter_list)
+        assert any("Male" in token for token in filter_list)
+        assert any("<dd>" in token for token in filter_list)
+        assert any("</dd>" in token for token in filter_list)
```

Comment:
With the `djlint` formatting of `list_officer.html` the spacing gets a little odd with regard to assertions like `<dd>{job.job_title}</dd>`. To deal with that, I broke up the assertions into three different statements.

Reply:
https://github.com/lucyparsons/OpenOversight/pull/966#discussion_r1260539806

Reply:
Same note here as in https://github.com/lucyparsons/OpenOversight/pull/966 smile

Reply:
I tried your solution and my solution of `filter_list = [token.replace(" ", "").replace("\n", "") for token in filter_list` to no avail.

The problem was that between the `<dd>` elements you had a starting line break and A LOT of spaces. To address that, I created this function and created all of the `filter_list`s with it.
```python
def normalize_tokens_for_comparison(html_str: str, split_str: str):
  """Remove new lines, leading, and closing spaces between <dd> elements in
  formatted HTML."""
  parsed_list = html_str.data.decode(ENCODING_UTF_8).split(split_str)[1:]
  parsed_list = [re.sub(r"<dd>\n\s+", "<dd>", token) for token in parsed_list]
  parsed_list = [re.sub(r"\n\s+</dd>", "</dd>", token) for token in parsed_list]
  return parsed_list
```

I've since added it to both branches.